spring-cloud-dataflow-server/docker-compose-prometheus.yml

```diff
@@ -38,13 +39,13 @@ services:
       - micrometer.prometheus-proxy.tcp-port=7001
 
   grafana:
-    image: springcloud/spring-cloud-dataflow-grafana-prometheus:${DATAFLOW_VERSION:?DATAFLOW_VERSION is not set! Use 'export DATAFLOW_VERSION=local-server-image-tag'}
+    image: springcloud/spring-cloud-dataflow-grafana-prometheus:${DATAFLOW_VERSION:-latest}
     container_name: grafana
     ports:
       - '3000:3000'
 
   prometheus:
-    image: springcloud/spring-cloud-dataflow-prometheus-local:${DATAFLOW_VERSION:?DATAFLOW_VERSION is not set! Use 'export DATAFLOW_VERSION=local-server-image-tag'}
```

Comment:
The `latest` is not a valid image tag nether for the SCDF or Skipper server nor for the Stream or Task apps.  An explicit app version tags is required.